modules/lunamediahbBidAdapter.md

```diff
@@ -0,0 +1,77 @@
+# Overview
+
+```
+Module Name: lunamedia Bidder Adapter
+Module Type: lunamedia Bidder Adapter
+Maintainer: support@lunamedia.io
+```
+
+# Description
+
+Module that connects to lunamedia demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+                // Will return static test banner
+                {
+                    code: 'adunit1',
+                    mediaTypes: {
+                        banner: {
+                            sizes: [[300, 250]],
+                        }
+                    },
+                    bids: [
+                        {
+                            bidder: 'lunamediahb',
+                            params: {
+                                placementId: 0
```

Comment:
please include working example placementIds so I can verify bids are returning properly through the adapter.

Reply:
I've added params from testing html file tha i'm using. Actualy placementId=0 should work with proper mediatype

Reply:
Banner and Video look good. 

Please fix Native, it is not passing your isBidResponseValid function because you are looking for 'native' property on the bid but you are returning an object with these keys: ["mediaType", "clickUrl", "title", "image", "impressionTrackers", "ttl", "cpm", "requestId", "creativeId", "netRevenue", "currency", "dealId", "meta"]

Reply:
I've added wrapping arround native assets, also i have noticed, that image and icon size now required and without them - native is invalid, so changed that to